### DISCUSSION TITLE
$refs don't refer to nested component

### DISCUSSION THREAD

**Discussion by schutm** [alpinejs/alpine#996] [4. General]
You can't refer to nested components by using their x-ref. E.g.

<div x-data>
  <span>a</span>
  <span x-data x-ref="b">b</span>
</div>

You can't refer to span b from the containing component using the $refs-syntax. Is this deliberate?

**Comment by KevinBatdorf** [comment]
That's intentional. See here: https://github.com/alpinejs/alpine/blob/master/src/component.js#L452
